### PR TITLE
Cypress/doc modularize

### PR DESCRIPTION
**Pull request by tobi-bams**
## Describe your changes
Added Documentation to help setup Cypress for hunter, Also made `login`, `logout` and `create_bounty` for hunter to reuse

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by Vayras** [comment]
@tobi-bams there is a test  written before `beforeEach(()=>...`  block copy the block above all of the tests
Screenshot 2024-02-02 at 10 03 29 AM

**Comment by Vayras** [comment]
you can either make this change or can wait till my PR with this fixed get merged, Thanks

**Comment by kevkevinpal** [comment]
just rebased to master  214a658

File: cypress/e2e/create_bounty.cy.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+describe('Alice tries to create a bounty', () => {
+  it('Create a bounty', () => {
+    let activeUser = 'alice';
+    cy.login(activeUser);
+    cy.wait(1000);
+
+    cy.create_bounty({
+      title: 'My new Bounty',
+      category: 'Web development',
+      // github_issue_url: 'https://github.com/stakwork/sphinx-relay/issues/799',
```
Comment (kevkevinpal) [inline cypress/e2e/create_bounty.cy.ts:10]: do we need these for later I'd rather not have commented code here

File: cypress/e2e/create_bounty.cy.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      coding_language: ['Typescript', 'Javascript', 'Lightning'],
+      description: 'This is available',
+      amount: '123',
+      assign: 'carol',
+      deliverables: 'We are good to go man',
+      tribe: '',
+      estimate_session_length: 'Less than 3 hour',
+      estimate_completion_date: '09/09/2024'
+      // organization: 'Testing Org'
```
Comment (kevkevinpal) [inline cypress/e2e/create_bounty.cy.ts:19]: do we need these for later I'd rather not have commented code here

**Comment by kevkevinpal** [comment]
This looks good to me, minus the two small comments nice work @tobi-bams 

if you can think of other common commands a user might want those might make good followup PR's

feel free to merge after the CI passes

**Comment by tobi-bams** [comment]
> This looks good to me, minus the two small comments nice work @tobi-bams
> 
> if you can think of other common commands a user might want those might make good followup PR's
> 
> feel free to merge after the CI passes

Thanks @kevkevinpal I left them there just as an example of what's possible, but I will remove them with my next PR.

**Pull request merged**